### ISSUE TITLE
Creating a sandbox with EmberJS works intermitently

### ISSUE THREAD

**Issue by lvegerano** [codesandbox/codesandbox-client#2202]
# 🐛 bug report

## Description of the problem
Initially forking a repo and then subsequently forking the sanbox works until I restart server/sandbox. Basically I get 404s when requesting the app.css and app.js files. They get sent with the wrong mime type.

## How has this issue affected you? What are you trying to accomplish?
Testing the waters to migrate to a paid version. 
I would like to show my coworkers the app.
I want to demo some new emberjs functionality.

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |modifiers
| Browser          | Chrome (latest)
| Operating System | MacOS

**Comment by lbogdan** [comment]
Hey @lvegerano ,

It looks like you have some errors in your sandbox:
```
$ ember serve
Could not start watchman
Visit https://ember-cli.com/user-guide/#watchman for moreinfo.

modifiers/templates/modifiers.hbs
  14:28  error  you must use double quotes in templates  quotes
  15:28  error  you must use double quotes in templates  quotes
  16:28  error  you must use double quotes in templates  quotes
  24:28  error  you must use double quotes in templates  quotes
  24:56  error  you must use double quotes in templates  quotes

===== 1 Template Linting Error


/sandbox/app/controllers/modifiers.js
   8:5   error  Unexpected console statement     no-console
  13:5   error  Unexpected console statement     no-console
  14:5   error  Unexpected console statement     no-console
  17:30  error  'evt' is defined but never used  no-unused-vars
  22:5   error  Unexpected console statement     no-console
  27:5   error  Unexpected console statement     no-console
  28:5   error  Unexpected console statement     no-console
  31:32  error  'evt' is defined but never used  no-unused-vars

✖ 8 problems (8 errors, 0 warnings)
```

**Comment by lvegerano** [comment]
Hi @lbogdan 

I fixed those, and I'm still having the issue.

**Comment by lbogdan** [comment]
Well, we're not really doing anything special in the container, just starting the devserver and proxying to it, so it's most probably an issue with the (state of the) project. Unfortunately I don't know much about Ember, so can't really help with that. 🙁

**Comment by sukima** [comment]
@lbogden I forked your linked project. There are some issues I found. The `package.json` says the name of the app is `modifiers` but the modulePrefix says `ember-3-11`. I changed the config and the `index.html` to match `modifiers` and the page loaded without any error.

What was happening was the Ember build was placing files to `dist/assets/modifiers.{js,css}` but the `index.html` was looking for `/assets/ember-3-11.{js,css}`. This results in a 404 from ember's server which is of content-type text/html.

After fixing those issues I was further confused as there was a blank page. Looked at the templates and the application./index template is blank. I then opened up `/modifiers` only to find a HBS compilation error: _Error: Compile Error: bothModifiers is not a helper_ for which I stopped debugging.

Cheers hope this helps.

@lvegerano **Confirmed** this is **not** a codesandbox issue.
